storage/container_storage_interface/persistent-storage-csi-ebs.adoc

```diff
@@ -12,12 +12,11 @@ toc::[]
 Familiarity with link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/[PVs], link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims[persistent volume claims (PVCs)], link:http://blog.kubernetes.io/2016/10/dynamic-provisioning-and-storage-in-kubernetes.html[dynamic
 provisioning], and link:https://kubernetes.io/docs/admin/authorization/rbac/[RBAC authorization] is recommended.
 
-Before PVCs can be created, you must install the AWS EBS CSI Driver Operator. The Operator provides a default StorageClass that you can use to create PVCs. You also have the option to create the EBS StorageClass as described in xref:../../storage/persistent_storage/persistent-storage-aws.adoc#persistent-storage-aws[Persistent Storage Using AWS Elastic Block Store].
+To create CSI-provisioned PVs that mount to AWS EBS storage assets, {product-title} installs the AWS EBS CSI Driver Operator and the AWS EBS CSI driver by default in the `openshift-cluster-csi-drivers` namespace.
```

Comment:
@jsafrane Do we need to suggest that the user upgrades the old Operator (if installed) before upgrading a cluster to allow an easier namespace change? (This was mentioned during design, though I might be misunderstanding the purpose and placement.)

Reply:
For the AWS EBS CSI Operator is a TP feature, so we will not support upgrade from 4.5 to 4.6, if the user has this operator installed on 4.5, he/she needs to uninstall the operator first. Need Jan to double confirm.

Reply:
If the user has a 4.5 OCP cluster, and he/she installed AWS EBS CSI Operator and Driver in this cluster, before the cluster is updated to 4.5, the AWS EBS CSI Operator and Driver need to be uninstalled.

Reply:
OK, added a note here that user must uninstall 4.5 Operator and driver before they can update to 4.6+.